front/src/components/Auth/ResetPassword.vue

```diff
@@ -4,14 +4,35 @@
       <q-card-section>
         <div class="text-h6">Reset password</div>
       </q-card-section>
-      <q-card-section class="q-gutter-md">
-        <password-input v-model="password" required />
-        <q-input filled readonly required :model-value="token" label="Token" />
+
+      <q-card-section class="q-pt-none q-pb-sm q-gutter-sm">
+        <password-input
+          v-model="password"
+          label="New password"
+          dense
+          required
+        />
+        <q-input
+          filled
+          dense
+          readonly
+          required
+          :model-value="token"
+          label="Password reset token"
+        >
```

Comment:
It annoys me a bit that the password reset token UUID does not fit by half a character. Maybe make the dialog a few pixels bigger?